src/CachePlugin.php

```diff
@@ -138,6 +137,40 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     }
 
     /**
+     * Calculate the timestamp when this cache item should be dropped from the cache. The lowest value return after
+     * the calculation is $maxAge.
+     *
+     * @param int|null $maxAge
+     *
+     * @return int|null Unix system time. @see PSR6 for caching
```

Comment:
sorry, `@see` should be on a separate line. maybe make this `@return int|null Unix system time passed to the PSR-6 cache`?